tests/Database/migrations/2020_01_30_0000_create_ingredients_table.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateIngredientsTable extends Migration
```

Comment:
Is there  a specific reason you added a new Model?

Reply:
I was trying not to mess with your existing setup and tests. In order to make sure that everything I was doing worked I'd have either had to delete the factory you created in your tests or create a new model. I don't mind either way. I can remove that and just work with the existing models instead. I ended up deleting doing a delete in my tests anyway when I noticed that the --force option was untested and didn't appear to work.

Reply:
ah ok I see. It's ok just leave 👍